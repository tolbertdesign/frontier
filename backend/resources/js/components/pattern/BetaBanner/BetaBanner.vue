<template>
  <div class="banner_dashboard mx-auto">
    <a
      href="/v3/tkdashboard/?redirect=home/dashboard?dashboardBetaOptOut=true"
      class="bg-grey"
    >
      <span class="message md:underline md:leading-normal leading-tight bg-grey text-white">
        {{ lang.beta_banner_return }}
      </span>
    </a>
  </div>
</template>

<script>
export default {
  computed: {
    lang () {
      return this.$store.state.lang
    },
  },
}
</script>
