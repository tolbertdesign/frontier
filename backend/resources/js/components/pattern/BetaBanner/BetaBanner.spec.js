import { shallowMount } from '@vue/test-utils'
import BetaBanner from './BetaBanner.vue'

describe('BetaBanner', () => {
  it('is a Vue instance', () => {
    const $store = {
      state: {
        lang: {
          beta_banner_return: 'Return',
        },
      },
    }

    const wrapper = shallowMount(BetaBanner, {
      mocks: { $store },
    })

    expect(wrapper.isVueInstance()).toBeTruthy()
    expect(wrapper).toMatchSnapshot()
  })
})
