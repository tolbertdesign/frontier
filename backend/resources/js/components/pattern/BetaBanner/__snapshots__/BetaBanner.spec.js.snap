// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BetaBanner is a Vue instance 1`] = `
<div class="banner_dashboard mx-auto"><a href="/v3/tkdashboard/?redirect=home/dashboard?dashboardBetaOptOut=true" class="bg-grey"><span class="message md:underline md:leading-normal leading-tight bg-grey text-white">
            Return
        </span></a></div>
`;
